src/app/vendors/Vendors.tsx

```diff
@@ -182,6 +184,29 @@ function Vendors({
     vendorGroups = filterVendorGroupsToSearch(vendorGroups, searchQuery, filterItems);
   }
 
+  if (
+    currentXur().start === undefined &&
+    vendorGroups &&
+    Object.values(vendorGroups).some((v) => v.def.hash === VENDOR_GROUPS.LIMITED_TIME)
```

Comment:
vendorGroups is already an array as far as i can tell, shouldn't need object.values